targets/zephyr/Makefile.zephyr

```diff
@@ -47,6 +47,11 @@ endif
 
 INTERM     = build/$(BOARD)/obj-$(BOARD)
 OUTPUT     = build/$(BOARD)
+DOTCONFIG  = $(OUTPUT)/zephyr/.config
+
+include $(DOTCONFIG)
```

Comment:
I admit, I was not aware of the implicit build target implications of `include`. It's OK then as it is.